src/logstash-filters/snippets/doppler.conf

```diff
@@ -0,0 +1,71 @@
+# Parse Cloud Foundry logs from doppler (via https://github.com/SpringerPE/firehose-to-syslog)
+
+json {
+    source => 'syslog_message'
+    add_tag => [ 'cloudfoundry_doppler' ]
+}
+
+if "cloudfoundry_doppler" in [tags] {
```

Comment:
Turns out the json filter doesn't have a "tag on failure" options.  However, since add_tag only happens if the json parsing is successful, I can use this (slightly weird) pattern to emulate this behaviour